src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -529,6 +529,10 @@
   /***********************************/                                                                                              \
                                                                                                                                      \
      static_field(StubRoutines,                _call_stub_return_address,                     address)                               \
+     static_field(StubRoutines,                _arraysort_int,                                address)                               \
+     static_field(StubRoutines,                _arraysort_long,                               address)                               \
+     static_field(StubRoutines,                _arraysort_float,                              address)                               \
+     static_field(StubRoutines,                _arraysort_double,                             address)                               \
```

Comment:
Should this be in hotspot/share/jvmci/vmStructs_jvmci.cpp instead?

Reply:
That's true. Moved it to hotspot/share/jvmci/vmStructs_jvmci.cpp. Thanks for catching this!